mgmtsystem_action/workflows/mgmtsystem_action.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-  <data noupdate="1">
+<odoo>
```

Comment:
+1

Maxime Chambreuil
Le 26 mai 2016 12:13, "Daniel Reis" <notifications@github.com> a écrit :

> In mgmtsystem_action/workflows/mgmtsystem_action.xml
> <https://github.com/OCA/management-system/pull/115#discussion_r64783842>:
>
> > @@ -1,6 +1,6 @@
> >  <?xml version="1.0" encoding="utf-8"?>
> > -<openerp>
> > -  <data noupdate="1">
> > +<odoo>
>
> Kill Workflows! Kanban replaces them. OK @max3903
> <https://github.com/max3903> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/management-system/pull/115/files/36cb8e6f85fcccd3f1824d530fbbe8021f2fb633#r64783842>
>